src/include/processor/operator/physical_operator.h

```diff
@@ -61,7 +62,7 @@ const string PhysicalOperatorTypeNames[] = {"AGGREGATE", "AGGREGATE_SCAN", "COLU
     "MULTIPLICITY_REDUCER", "PROJECTION", "SCAN_REL_PROPERTY", "RESULT_COLLECTOR", "SCAN_NODE_ID",
     "SCAN_COLUMN_PROPERTY", "SEMI_MASKER", "SET_STRUCTURED_NODE_PROPERTY", "SKIP", "ORDER_BY",
     "ORDER_BY_MERGE", "ORDER_BY_SCAN", "UNION_ALL_SCAN", "UNWIND", "VAR_LENGTH_ADJ_LIST_EXTEND",
-    "VAR_LENGTH_COLUMN_EXTEND"};
+    "VAR_LENGTH_COLUMN_EXTEND", "SHORTEST_PATH"};
```

Comment:
Let's be consistent with (var length) extend operators. Separate SHORTEST_PATH to COLUMN_SHORTEST_PATH, and ASJ_LIST_SHORTEST_PATH.

Reply:
Done